waagent

```diff
@@ -5876,7 +5880,7 @@ def main():
     if len(sys.argv) == 1:
         sys.exit(Usage())
 
-    LoggerInit('/var/log/waagent.log','/dev/console')
+    LoggerInit('/dev/stdout',None)
```

Comment:
I see. It make sense to disable it in docker container. However, would you please change the default behavior to 'y'. Otherwise, when existing user upgrade agent to this new version, they would lost log.